aicssegmentation/structure_wrapper/WorkflowEngine.py

```diff
@@ -25,6 +26,15 @@ def __init__(self, workflow_name: str, image: np.ndarray):
             image (np.ndarray):  image to perform workflow on
         """
         self.workflow_name: str = workflow_name  # Workflow name
+
+        self.widget_info: Dict[str, Any] = None
+        with open(
+            Path(__file__).parent.parent
+            / "structure_wrapper_config"
+            / "all_functions.json"
+        ) as file:
+            self.widget_info = json.load(file)
```

Comment:
Why is this called `widget_info`? This looks like it's loading all the functions information, not just widget information? It's confusing to me. Would something like `function_info` be more clear?

Reply:
i'll name this all_function_info.